docs/arch.md

```diff
@@ -138,6 +144,26 @@ health_statuses_insights{metric="moderate"} 1
 health_statuses_insights{metric="total"} 2
 ```
 
+### Metrics
+
+- `health_statuses_insights`, information about the cluster health status as detected by Insights tooling
+- `insightsclient_request_send_total`, tracks the number of metrics sends.
+- `insightsclient_request_recvreport_total`, tracks the number of reports requested.
```

Comment:
> tracks the number of reports requested.
I think it's not exactly accurate. It's number of received/downloaded Insights reports.

Reply:
Same thing here: 
https://github.com/openshift/insights-operator/blob/0929403ce543afd46074ee9b3dc20de8ef01d069/pkg/insights/insightsclient/insightsclient.go#L273

Reply:
Yeah same here....We request the report in fact, but this metric is incremented only when it's downloaded AFAIK